rocky/rocky/views/organization_member_list.py

```diff
@@ -25,15 +26,28 @@ class OrganizationMemberListView(
     ListView,
 ):
     model = OrganizationMember
-    filters_active: List[str] = []
     context_object_name = "members"
 
     def get_queryset(self):
-        return self.model.objects.filter(organization=self.organization)
+        queryset = self.model.objects.filter(organization=self.organization)
+        if "blocked_status_filter" in self.request.GET:
+            blocked_status_filter = self.request.GET.getlist("blocked_status_filter", [])
+            queryset = self.filter_queryset(queryset, blocked_status_filter)
+        for member in queryset:
+            if member.status == "blocked":
+                member.blocked = True
+        return queryset
+
+    def filter_queryset(self, queryset, blocked_status_filter):
+        result = []
+        if "blocked" in blocked_status_filter:
+            result.extend([member for member in queryset if member.status == "blocked"])
+        if "unblocked" in blocked_status_filter:
+            result.extend([member for member in queryset if not member.status == "blocked"])
+        return result
```

Comment:
Does this work?
```suggestion
        return [member for member in queryset if member.status in blocked_status_filter]
```

Reply:
I think this doesn't work because `unblocked` doesn't exist in `member.status`. We could opt for just a status filter, instead of a "blocked_status_filter". Then it should work, I guess.

Reply:
`...a status filter, instead of a "blocked_status_filter"`

Done just that.